rspace/src/test/scala/coop/rchain/rspace/HotStoreSpec.scala

```diff
@@ -850,10 +851,14 @@ trait HotStoreSpec[F[_], M[_]] extends FlatSpec with Matchers with GeneratorDriv
       res: Either[Throwable, Unit],
       actual: Seq[T],
       initial: Seq[T],
-      index: Int
+      index: Int,
```

Comment:
This function becomes even more unreadable. The name `checkRemoval` doesn't say much about what it does.